scripts/new-change

```diff
@@ -53,7 +53,12 @@ CHANGES_DIR = os.path.join(
     '.changes'
 )
 TEMPLATE = """\
-# Type should be one of: feature, bugfix
+# Type should be one of: feature, bugfix, enhancement, api-change
+# feature: A larger feature or change in behavior, usually resulting in a
+#          minor version bump
```

Comment:
Sorry. Super nit picky but probably add a period at the end to make it consistent with the others. Or you can remove the periods.